src/map/skill.cpp

```diff
@@ -14688,6 +14787,13 @@ int skill_castend_pos2(struct block_list* src, int x, int y, uint16 skill_id, ui
 		}
 		break;
 
+	case SOA_TALISMAN_OF_BLACK_TORTOISE:
+		if (sc && (sc->getSCE(SC_T_THIRD_GOD) && !sc->getSCE(SC_T_FOURTH_GOD) && !sc->getSCE(SC_T_FIFTH_GOD))){
```

Comment:
```suggestion
		if (sc && sc->getSCE(SC_T_THIRD_GOD)){
```

According to divine-pride the status will be changed without a condition:

If the user has [Blessing of the South] buff, The buff will be changed to [Blessing of the North].

https://divine-pride.net/database/skill/5429